siphon/simplewebservice/igra2.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
+# Distributed under the terms of the MIT License.
+# SPDX-License-Identifier: MIT
+"""Read upper air data from the Integrated Global Radiosonde Archive version 2."""
+
+import datetime
+from io import BytesIO
+from io import StringIO
+import itertools
+import sys
+from urllib.request import urlopen
```

Comment:
If you replace with:
```python
from ..http_util import urlopen
```
that will take care of the Python 3 issue and all requests will have Siphon's user agent string.

Reply:
I tried swapping urllib.request for the http_util, but am running into an issue with the fact that I'm downloading FTP instead of HTTP:
```InvalidSchema: No connection adapters were found for 'ftp://ftp.ncdc.noaa.gov/pub/data/igra/data/data-por/USM00070026-data.txt.zip'```

I'm looking for another method.

Reply:
For now I just put in a try/except:
````    
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

Reply:
That's fine with me. If it doesn't work, then our own `urlopen` doesn't really count as a "helper". 😄